src/core/reactor_backend.cc

```diff
@@ -550,6 +560,10 @@ class aio_pollable_fd_state : public pollable_fd_state {
     future<> get_completion_future(int events) {
         return get_desc(events)->get_future();
     }
+    void abort(std::optional<std::exception_ptr> ex = std::nullopt) noexcept {
+        get_desc(POLLIN)->abort(ex);
+        get_desc(POLLOUT)->abort(ex);
+    }
```

Comment:
I guess, `aio_pollable_fd_state::get_desc` and `epoll_pollable_fd_state::get_desc` get different 'enum' for argument, one takes: POLLIN, POLLOUT, the other one takes EPOLLIN, EPOLLOUT. This may be a bit wrong to mix them like single entity.